### ISSUE TITLE
Add babel-core as dev dependency

### ISSUE THREAD

**Issue by piotr-cz** [rauschma/webpack-babel-demo#2]
When running `npm run build` I'm getting _ERROR in Cannot find module 'babel-core'_
Installing package resolves issue

**Comment by yukk1** [comment]
You should install babel-core.
`npm  install babel-core --save-dev`

**Comment by piotr-cz** [comment]
No, it should be specified in _package.json devDependencies_ and installed on `npm install`
